### ISSUE TITLE
[Bug] "about:crashes" page not doing its job

### ISSUE THREAD

**Issue by concastinator** [mozilla-mobile/fenix#8769]
## Steps to reproduce

I am going to assume that Firefox Preview 3 or Nightly has crashed at least once on your system. Preferably, it has automatically sent a crash report. Open the about:crashes page in the address bar.

### Expected behavior

Judging by the name of the page, you should see a list crashes and/or of (un-) submitted crash reports.

### Actual behavior

You will only see the text "No crash reports have been submitted".

### Device information

* Android device: Nokia 5 (model TA-1024), Android 9.
* Fenix version: all of them, it seems; both Nightly and Preview 3.

**Comment by cadeyrn** [comment]
about:crashes is not supported in GV apps. #2295 is a request to implement something like about:crashes in Firefox Preview.

**Comment by pocmo** [comment]
It landed in Fenix and is available via "Settings > About > Crashes". For now it also opens if you type `about:crashes` into the toolbar (but not for other ways of navigating to that URL).

**Comment by skierpage** [comment]
> It landed in Fenix and is available via "Settings > About > Crashes". For now it also opens if you type `about:crashes` into the toolbar (but not for other ways of navigating to that URL).

After several crashes today on my Pixel 3 I entered the location about:crashes and saw the old-school "Submitted Crash Reports" with only one crash report from back in May 2020. Then I tried Settings > About Firefox > Crashes and saw the new-style crash list, and when I visited about:crashes again, I saw the same new-style crash list. Weird. I could not reproduce this on a Lenovo Yoga Tab 3 Pro.
